content/home/halfPrice/halfPrice.md

```diff
@@ -1,12 +1,12 @@
 ---
-heading: "S3-compatible and Half the Price"
+heading: "Straight-Forwarding Pricing"
 col1Heading: "Storage"
 col1Image: "/img/tar-ico-cloud.png"
-col1Amount: "$10.00"
-col1Metric: "Per TB Per Month"
+col1Amount: "$0.01"
+col1Metric: "PER GB PER MONTH"
 col2Heading: "Bandwidth"
 col2Image: "/img/tar-ico-arrows-left-right.png"
-col2Amount: "$45.00"
-col2Metric: "Per TB Of Outbound Data"
-cta: "Get Started"
+col2Amount: "$0.045"
+col2Metric: "PER GB PER MONTH"
```

Comment:
here should be just GB

According to my last comment you should have fixed only storage dimension to GB - Month. Bandwidth was ok, just GB